package.json

```diff
@@ -8,16 +8,18 @@
   },
   "devDependencies": {
     "casperjs": "~1.1.0-beta3",
+    "jscs": "~2.6.0",
+    "jshint": "~2.9.1",
```

Comment:
If I wipe out my node_modules and run any of the js make targets, it happily installs jshint 2.9.1-rc1 for me. My understanding of the '~' in the version specifier is that it will satisfy it with anything matching the minor version. Ie, like '2.9.x'.